chaosblade-exec-common/src/main/java/com/alibaba/chaosblade/exec/common/model/action/exception/DefaultThrowExceptionExecutor.java

```diff
@@ -62,8 +62,14 @@ public Exception throwCustomException(ClassLoader classLoader, String exception)
         try {
             Class<?> clazz = classLoader.loadClass(exception);
             if (Exception.class.isAssignableFrom(clazz)) {
-                Constructor<?> constructor = clazz.getConstructor(String.class);
-                return (Exception)constructor.newInstance("chaosblade-mock-exception");
+            	Constructor<?>[] constructors = clazz.getConstructors();
+    			for(Constructor<?> constructor: constructors) {
+    				if(constructor.getParameterCount() == 0) 
```

Comment:
`getParameterCount()` method was added since 1.8 JDK version, but the agent build version is 1.6, so the method can't be used. We can use `getParameterTypes()` method to get the constructor parameters.

Reply:
Oh!  That's right, I'll refactor the code later.